src/components/motor/MotorController.h

```diff
@@ -7,19 +7,48 @@
 namespace Pinetime {
   namespace Controllers {
     static constexpr uint8_t pinMotor = 16;
-
+    
+    
     class MotorController {
+    
+
     public:
+      enum TuneType : uint8_t {
+        NOTIFICATION,
+        SHORT,
+        RING,
+        STOP
```

Comment:
STOP should probably be first if it is needed at all.

Reply:
STOP is needed for the RunForDuration functionality. RunForDuration turns on the motor and then schedules a STOP event at the appropriate time.

Reply:
I know but with some rework it might not be necessary. Or maybe it is. Just saying.

Reply:
removed STOP state